rosetta-lang/src/main/java/com/regnosys/rosetta/validation/RosettaSimpleValidator.xtend

```diff
@@ -1134,7 +1138,7 @@ class RosettaSimpleValidator extends AbstractDeclarativeRosettaValidator {
 			if (cardinality.isMulti(arg)) {
 				error('''The argument of «ele.operator» should be of singular cardinality.''', ele, ROSETTA_UNARY_OPERATION__ARGUMENT)
 			}
-			if (!arg.RType.isSubtypeOf(UNCONSTRAINED_STRING)) {
+			if (!arg.RMetaAnnotatedType.isSubtypeOf(UNCONSTRAINED_STRING)) {
```

Comment:
I think we should add shortcuts such as `UNCONSTRAINED_STRING_WITH_NO_META` etc

Reply:
done